.github/workflows/future-release-branch-labeler.yml

```diff
@@ -0,0 +1,43 @@
+name: Future Release Branch Pull Request Labeler
+
+on:
+  pull_request:
```

Comment:
As this workflow intends to use a secret, I recommend triggering on `pull_request_target`:
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target

Reply:
thanks for the feedback! it's been updated :D